documentation/pdb_client_tools.markdown

```diff
@@ -0,0 +1,79 @@
+---
+title: "PuppetDB 4.0: PuppetDB CLI"
+layout: default
+canonical: "/puppetdb/latest/pdb_client_tools.html"
```

Comment:
Apparently, this is now optional: https://github.com/puppetlabs/puppetdb/commit/67606cbb07c91e45efbfbe8d774be028c938c666